docs/integrations/slurm-integration.md

```diff
@@ -0,0 +1,99 @@
+# Slurm Integration Guide
+
+The AMD GPU Metrics Exporter provides integration with Slurm workload manager to track GPU metrics for Slurm jobs. This guide explains how to set up and configure this integration.
+
+## Prerequisites
+
+- Slurm workload manager installed and configured
+- AMD GPU Metrics Exporter installed and running
+- Root or sudo access on Slurm nodes
+
+## Installation
+
+- Copy the integration script:
+
+```bash
+sudo cp /usr/local/etc/metrics/slurm/slurm-exporter.sh /etc/slurm/
+sudo chmod +x /etc/slurm/slurm-exporter.sh
+```
+
+- Configure Slurm:
+
+```bash
+sudo vi /etc/slurm/slurm.conf
+
+# Add these lines:
+prologFlags=Alloc
+Prolog=/etc/slurm/slurm-exporter.sh
```

Comment:
The slurm script is not correct.

Reply:
@AMD-melliott  could you please compare the content with the content in the following standalone instllation guide?
https://amdcloud.sharepoint.com/:w:/s/DatacenterGPUProdMScottDirects/Ef-BiGoZ1tVMqAc1somFQDsBaN16pOU4w9XKcETsYnWP1w?e=0XNIAF

Reply:
I went through the guide again and migrated the relevant info. I excluded the Debian installer method since that is not yet available, and updated the parts that were outdated.